src/bin/default.js

```diff
@@ -1,6 +1,12 @@
 import path from 'path';
 import {isCI} from '../lib/ci';
 
+const chromedriver = require('../../../../package').devDependencies.chromedriver;
```

Comment:
that seems problematic, maybe something like https://www.npmjs.com/package/find-package-json . It uses cwd() by default so it should find a proper package.json in most cases.